### PR TITLE
Projects/kotlin/content update

### PR DESCRIPTION
**Pull request by bentley-ritson-umuzi**

### CODE REVIEW

File: content/syllabuses/ios-mobile.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: iOS Mobile Development
+---
+
+## Course Content Link
+- {{% contentlink "https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6" %}}
```
Comment (ry-oc) [inline content/syllabuses/ios-mobile.md:6]: Should be:
- [Stanford CS193P iOS11 Swift4](https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6)

File: content/syllabuses/ios-mobile.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Part 1
+
+- Topic : iOS Overview (Lecture 1)
+- Topic : MVC (Lecture 2)
+- Topic : Swift Development and other Tools (Lecture 3)
+- {{% contentlink "projects/ios-mobile/swift-and-more" %}}
```
Comment (ry-oc) [inline content/syllabuses/ios-mobile.md:13]: This page does not exist so it renders as 'PROJECTS: [MISSING title or page]'

File: content/syllabuses/java.md [hunk 3]
```diff
@@ -2,6 +2,49 @@
 title: Java Basics 
 ---
 
+ Introduction to NCIT curriculum
```
Comment (ry-oc) [inline content/syllabuses/java.md:5]: Missing NCIT:
5, 8, 11, 13, 14, 15, 16, 17, 20

File: content/projects/kotlin/liner-layout-using-the-layout-editor/_index.md [hunk 4]
```diff
@@ -0,0 +1,495 @@
+---
+title:  Linear layout using the Layout Editor
+ready: True
+prerequisites:
+  hard: ["topics/kotlin/grid-layout", "topics/kotlin/basic-control-flow", "topics/kotlin/basic-syntax-types"]
```
Comment (ry-oc) [inline content/projects/kotlin/liner-layout-using-the-layout-editor/_index.md:5]: topics/kotlin/grid-layout does not exist here so the content link does not work but it may be in another PR?

Comment (bentley-ritson-umuzi) [reply content/projects/kotlin/liner-layout-using-the-layout-editor/_index.md:5]: Yes it is in the andriod conternt one

File: content/syllabuses/java.md (hunk 3 above)
Comment (bentley-ritson-umuzi) [reply content/syllabuses/java.md:5]: Yes i will add those later. but i need the specialization stuff done first.

File: content/syllabuses/ios-mobile.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: iOS Mobile Development
+---
+
+## Course Content Link
+
+[Stanford CS193P iOS11 Swift4]("https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6")
```
Comment (ry-oc) [inline content/syllabuses/ios-mobile.md]: the " should be removed or it doesnt render as al link
